foundation_cms/templates/patterns/blocks/themes/default/portrait_card.html

```diff
@@ -9,9 +9,7 @@
 {% endif %}
 
 <div class="portrait-card__text">
-  {% if card.label %}
-    <p class="portrait-card__label label-text-regular">{{ card.label }}</p>
-  {% endif %}
+  <p class="portrait-card__label label-text-regular">{{ card.label }}</p>
```

Comment:
@robdivincenzo With this approach we will have empty `<p></p>` in the DOM when `card.label` is not present. What do you think about adjusting the top padding of `portrait-card__text` and/or the top padding of `portrait-card__headline` instead?